pkg/sr/api.go

```diff
@@ -150,12 +157,12 @@ func (cl *Client) Subjects(ctx context.Context, deleted HideShowDeleted) ([]stri
 //
 //	{"type":"boolean"}
 func (cl *Client) SchemaTextByID(ctx context.Context, id int) (string, error) {
-	// GET /schemas/ids/{id}
-	var s Schema
-	if err := cl.get(ctx, fmt.Sprintf("/schemas/ids/%d", id), &s); err != nil {
+	// GET /schemas/ids/{id}/schema
+	var s []byte
```

Comment:
Does this work? When the unmarshal destination is a []byte, the json package auto-base64-decodes. Doesn't this need to be `string`?

Reply:
Yeah I added it as a utility to get the raw body if needed, like here for example. I mean the alternative would be to leave the method as it is and go through the other endpoint that returns a JSON.

Reply:
@twmb any chance we could get this merged? I see a failing CI action, but it seems that it's not related to the code changes and was caused by the redpanda container not stopping correctly.

Reply:
Ah I didn't realize I left this unmergd.